tests/cases/conformance/salsa/plainJSGrammarErrors.ts

```diff
@@ -208,5 +208,3 @@ function foo() { new.targe }
 const nullaryDynamicImport = import()
 const trinaryDynamicImport = import('1', '2', '3')
 const spreadDynamicImport = import(...[])
-
-return
```

Comment:
Should we be leaving this return here so that we don't accidentally readd this error?